cwltool/extensions-v1.1.yml

```diff
@@ -44,3 +44,23 @@ $graph:
         subscope: run
       doc: |
         Specifies the process to run.
+
+- name: MPIRequirement
```

Comment:
I think this extension could work against `v1.0` files as well ; is there a particular reason you chose v1.1 as the starting point?

Reply:
Cos it's newer and I only wanted to write the thing once while it was under development? Added to 1.0 extensions also.